examples/sparse/sign.py

```diff
@@ -0,0 +1,93 @@
+"""
```

Comment:
Let's adopt the similar style (model definition + train method) as we established in https://github.com/dmlc/dgl/tree/master/tutorials/blitz.